modelgauge/main.py

```diff
@@ -228,6 +241,81 @@ def run_test(
     print("Full TestRecord json written to", output_file)
 
 
+@modelgauge_cli.command()
+@click.option(
+    "sut_names",
+    "-s",
+    "--sut",
+    help="Which registered SUT(s) to run.",
+    multiple=True,
+    required=True,
+)
+@click.option(
+    "--workers",
+    type=int,
+    default=None,
+    help="Number of worker threads, default is 10 * number of SUTs.",
+)
+@click.option(
+    "--debug", is_flag=True, help="Show internal pipeline debugging information."
+)
+@click.argument(
+    "filename",
+    type=click.Path(exists=True),
+)
+def run_prompts(sut_names, workers, filename, debug):
+    """Take a CSV of prompts and run them through SUTs. The CSV file must have UID and Text columns, and may have others."""
+
+    load_plugins()
+    secrets = load_secrets_from_config()
+
+    try:
+        suts = {
+            sut_name: SUTS.make_instance(sut_name, secrets=secrets)
+            for sut_name in sut_names
+        }
+    except MissingSecretValues as e:
+        raise_if_missing_from_config([e])
+
```

Comment:
It might be helpful to check that the SUTs all have the `AcceptsTextPrompt` capability before starting the pipeline.

Reply:
This was surprisingly hard to test, but I believe I now have this.

Reply:
Looks good, thanks!